frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -1015,8 +1015,37 @@ initiateCrossChainTransfer model networkName envs publicMeta keys fromAccount fr
       , _pmSender = senderText
       }
 
--- | Listen to a request key for some continuation
-listenForContinuation
+packHttpErrors :: MonadIO m => Logger t -> [S.ClientError] -> m (Either Text a)
+packHttpErrors logL es = do
+  let prettyErr = T.unlines $ fmap (tshow . packHttpErr) es
+  putLog logL LevelWarn prettyErr
+  pure $ Left prettyErr
+
+-- |Poll a request key for the second step (redeeming coin on target chain)
+pollNodesForSuccess
+  :: ( TriggerEvent t m
+     , PerformEvent t m
+     , MonadJSM (Performable m)
+     , HasLogger model t
+     )
+  => model
+  -> [S.ClientEnv]
+  -> Event t (Pact.PactId, Pact.RequestKey)
+  -> m (Event t (Either Text ()))
+pollNodesForSuccess model envs reqEv = pollNodesForReq model envs (first Just <$> reqEv) 12 15 $ \pr mpid->
```

Comment:
I think we should probably increase the max number of polls a good bit.  Three minutes does seem like a reasonable estimate but I think we should build in more of a buffer.  I'm thinking double it to 24...i.e. 6 minutes.